package com.sece.eceb.Controllers;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HelloController {

    @GetMapping("/")
    public String helloWorld(@RequestParam String name) {
        return "Hello World " + name;
    }

    @PostMapping("/")
    public String helloWorldPost(@RequestParam String name) {
        return "Account created for " + name;
    }
}